src/expr/skolem_manager.h

```diff
@@ -452,6 +449,16 @@ class SkolemManager
    * @return n in original form.
    */
   static Node getOriginalForm(Node n);
+  /**
+   * Convert to unpurified form, which returns the term that k purifies. In
```

Comment:
This documentation seems a bit misleading, since there is no actual computation being done to do the convertion. I think it'd be good to clarify that the unpurified forms are static and set up during the creation of the skolems.